.github/workflows/build-php-storage-driver-common.yml

```diff
@@ -0,0 +1,72 @@
+on:
+  workflow_call:
+    inputs:
+      hasCodeChanged:
+        required: true
+        type: boolean
+      isTag:
+        required: true
+        type: boolean
+      BUILD_AWS_ACCESS_KEY_ID:
+        type: string
+        required: true
+    secrets:
+      BUILD_AWS_SECRET_ACCESS_KEY:
+        required: true
+      S3_BUCKET:
+        required: true
+      S3_AWS_REGION:
+        required: true
+
+concurrency: build-php-storage-driver-common
+
+jobs:
+  build_common_driver:
+    if: ${{ inputs.hasCodeChanged || inputs.isTag }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v1
+
+      - name: Build Image
+        env:
+          DOCKER_BUILDKIT: 1
+          AWS_ACCESS_KEY_ID: ${{ inputs.BUILD_AWS_ACCESS_KEY_ID }}
```

Comment:
pro pristup aws by chtelo pouzivat role

Reply:
Diky, zalozim na to task mame teraz vsetky libky spravene a chcelo by to asi upravit

Reply:
https://keboola.atlassian.net/browse/CT-883

Reply:
Jo v CI accountech by idealne uz zadni IAM useri byt nemeli. Rad s tim pomuzu cokoliv resite s AWS davejte mi prosim vedet.

Reply:
Ted koukam ze je to dokonce v legacy dev AWS accountu. Melo by byt v `CI-Connection-Team`

Reply:
@Halama mame prosim ta niekde napisany ten postup? Ze by som to skusil rovno spravit ?

Reply:
Jo tady https://keboola.atlassian.net/wiki/spaces/KB/pages/2623209501/AWS+Azure+OIDC+authorization+in+GitHub+Actions a jsou tam linky na vsechny pouziti. Ted to asi nech takhle a zavolejme si nad tim treba ke konci tydne a dotahneme to samostatne. Hod kdyztak prosim do kalendare nejak.

Reply:
Ok dovtedy si to prejdem aspon :)

Reply:
je to super jednoduchy jenom se meni zpusob jakym se vygenerujou credentials, jinak se nic nemeni. Jenom u konkretniho repa promyslime jak tu roli nahodime a jak to pripadne bude nahazovat AWS resources ktere pro ty testy potrebuje.